.codespell.exclude

```diff
@@ -30,6 +30,7 @@
 		"rasi it maka makpes"                   # fish en papillote
 		"Far Upugf Enrel Ugt"                 # Nam Myoho Renge Kyo (Buddhist religious chant)
 		"Far Uhudf Elrep Uds"                  # Nam Myoho Renge Kyo (Buddhist religious chant)
+			"Capitalise that proper noun. Use American English for consistency. For goodness' sake, that needs a comma!"
```

Comment:
I wrote it as if I was writing and speaking it normally – to me that comma is optional, but I'd pause there if I said it out loud. The American English bit is just supposed to be juxtaposition. All my PRs have some commit about "stupid american english".

We don't _need_ to make a joke out of every bit in the sentence, do we? It's just about how I review.

Reply:
As per bene's statement, I was thinking of this as a one off thing, rather than a full endorsement. I'm actually expecting someone in the far future to actually make a review to revise this, but that's something to look forward to as an in joke until someone notices.

Reply:
If the intention is to make it a play on words, I'd say lean into it like Amazinite says so it's more flavorful.,

If the intention is a reference to bene's review style, then it probably should stay that way.